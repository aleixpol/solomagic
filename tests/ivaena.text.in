\ref something0
\mt somethingsomething ivena somethingsomething

\ref something1
\mt somethingsomething ivenang somethingsomething
