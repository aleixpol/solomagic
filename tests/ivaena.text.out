\ref something0
\mt somethingsomething ivena somethingsomething
\ma somethingsomething iva ena somethingsomething

\ref something1
\mt somethingsomething ivenang somethingsomething
\ma somethingsomething ivenang somethingsomething
